package.json

```diff
@@ -22,24 +22,25 @@
     "generate:ts-from-yaml:nodeNm3psp": "shx mkdir -p generated/nodeNm3psp && rimraf generated/nodeNm3psp/* && gen-api-models --strict 0 --api-spec generated/yaml-file/nodeForPsp_Service.yaml --out-dir generated/nodeNm3psp",
     "generate:ts-from-yaml:nodeNm3io": "shx mkdir -p generated/nodeNm3io && rimraf generated/nodeNm3io/* && gen-api-models --strict 0 --api-spec generated/yaml-file/nodeForIO_Service.yaml --out-dir generated/nodeNm3io",
     "generate:yaml-from-wsdl": "shx mkdir -p generated/yaml-file && rimraf generated/yaml-file/* && node pagopa_api/YamlGenerator.js",
-    "generate:proxy-api-models": "shx mkdir -p generated/api && rimraf generated/api/* && gen-api-models --api-spec ./api_pagopa.yaml --out-dir generated/api --ts-spec-file generated/api/public_api_pagopa.ts --request-types",
+    "generate:proxy-api-models": "shx mkdir -p generated/api && rimraf generated/api/* && gen-api-models --api-spec ./api_pagopa.yaml --out-dir generated/api --request-types",
     "prettify": "prettier --write \"./**/*.ts\"",
     "start": "node dist/src/index.js",
-    "test:coverage": "jest --runInBand --coverage",
-    "test:integration": "jest --config=jest.integration.js --runInBand",
+    "test:coverage": "jest --runInBand --coverage --detectOpenHandles",
```

Comment:
with `--detectOpenHandles` option , when i runs  test i retrieve the following nasty warning message
```sh
Jest has detected the following 1 open handle potentially keeping Jest from exiting:

  ●  TCPWRAP
```

is it correct to display it? or is there a way to disable it?

Reply:
added for some debugging, but we can disable, thanks